### ISSUE TITLE
[BUG] If the cert bundle is prepared or opened in windows platform then carriage return cause issue.

### ISSUE THREAD

**Issue by praveenkumar** [code-ready/crc#1957]
Recently a user #1956 from this bug tried the crc with proxy and getting following error which means the carriage return is not properly handled. Mostly https://github.com/code-ready/crc/blob/master/pkg/crc/cluster/cluster.go#L218 logic need to be changed. 

```
Failed to update cluster proxy configuration: Failed to add proxy cert details ssh command error:
command : oc apply -f /tmp/user-ca-bundle.json --context admin --cluster crc --kubeconfig /opt/kubeconfig
err : Process exited with status 1\n: error: error parsing /tmp/user-ca-bundle.json: json: line 3: invalid character '\r' in string literal
```

**Comment by cfergeau** [comment]
How did the user got into that situation though? Did they edit the file manually? Was the file provided to them this way?
On Windows, we could indeed strip all \r from text files we are sending to the VM.

**Comment by praveenkumar** [comment]
> How did the user get into that situation though? Did they edit the file manually? Was the file provided to them this way?

@cfergeau my assumption this is how it was provided or may be copy pasted using notepad which auto add those carriage return.

> On Windows, we could indeed strip all \r from text files we are sending to the VM.
Yes, but here we are changing the carriage return to char.

**Comment by vyom-soft** [comment]
I can confirm nothing is editied manually not it is open in any editor. The same cert works for 4.6.6.

**Comment by praveenkumar** [comment]
resolved by #1958

**Comment by vyom-soft** [comment]
> resolved by #1958

@praveenkumar, **Great to know that it has been resolved**. So if I download the release will it also have the fix available...
